hermes.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Author: Evan Spendlove (@evanSpendlove)
+//
+// Main program loop for Hermes. This initialises Cloudprober so that Hermes can
+// interact with it through gRPCs.
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"strconv"
+
+	"github.com/golang/glog"
+	"github.com/google/cloudprober"
+	"github.com/google/cloudprober/web"
+	"github.com/googleinterns/step224-2020/hermes"
+)
+
+var (
+	rpc_port = flag.Int("rpc_port", 9314, "The port that the gRPC server of Cloudprober will run on")
+)
+
+func main() {
+	flag.Parse()
+
+	hermes := &hermes.Hermes{}
+	err := hermes.InitialiseCloudproberFromConfig(buildConfig())
+	if err != nil {
+		glog.Exitf("cloudprober could not be initialised from config: grpc_port: %d, err:%v", *rpc_port, err)
+	}
+
+	// Sets up web UI for cloudprober.
+	web.Init()
+
+	cloudprober.Start(hermes.Ctx)
+
+	// Wait forever
+	select {}
+}
+
+// buildConfig() builds the configuration details for Cloudprober based on the flag contents.
+// Returns:
+// - string: Returns the configuration details for Cloudprober as a string.
+func buildConfig() string {
+	return fmt.Sprintf("grpc_port: " + strconv.Itoa(*rpc_port))
```

Comment:
You don't need to use the `+` operator to concatenate strings.
```suggestion
	return fmt.Sprintf("grpc_port: %d", *rpc_port)
```

Reply:
Ack.